rrfs-test/validated_yamls/templates/obtype_config/adpsfc_winds_287.yaml

```diff
@@ -0,0 +1,213 @@
+
+       obs error:
+         covariance model: diagonal
+
+       obs localizations:
+         - localization method: Horizontal Gaspari-Cohn
+           lengthscale: 300e3 # orig
+
+       obs filters:
+         # ------------------
+         # wind (287)
+         # ------------------
+         # Reject all obs with ObsType /= 287, QualityMarker = 4 -15
+         - filter: RejectList
+           apply at iterations: 0,1
+           where:
+           - variable: ObsType/windEastward
+             is_not_in: 287
+         - filter: RejectList
+           where:
+           - variable: QualityMarker/windEastward
+             is_in: 4-15
+
+         # Time window filter
+         - filter: Domain Check
+           apply at iterations: 0,1
+           where:
+           - variable:
+               name: MetaData/timeOffset
+             minvalue: -900
+             maxvalue:  900
+
+         # Temporal thinning filter
+         - filter: Temporal Thinning
+           apply at iterations: 0,1
+           min_spacing: PT30M
+           seed_time: *analysisDate
+           category_variable:
+             name: MetaData/stationIdentification
+
+         ## Online domain check
+         #- filter: Bounds Check
+         #  filter variables:
+         #  - name: specificHumidity
+         #  test variables:
+         #  - name:  GeoVaLs/observable_domain_mask
+         #  minvalue: 0.0
+         #  maxvalue: 0.5
+
+         # Initial error assignment
+         - filter: Perform Action
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           where:
+           - variable:
+               name: ObsType/windEastward
+               is_in: 287
+           action:
+             name: assign error
+             error function:
+               name: ObsFunction/ObsErrorModelStepwiseLinear
+               options:
+                 xvar:
+                   name: MetaData/pressure
+                 xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
+                 errors: [1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874, 1.5874]
+
+         # error inflation based on pressure check (setupw.f90)
+         - filter: Perform Action
+           filter variables:
+           - name: windEastward
+           where:
+           - variable: ObsType/windEastward
+             is_in: 287
+           action:
+             name: inflate error
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windEastward
+                 inflation factor: 4.0
```

Comment:
Sorry if I missed it, but there should be two ObsErrorFactorPressureCheck's for the winds yamls: one for windEastward and one for windNorthward. There are also some new options that I recently added. The section should look like this:
```
         # Error inflation based on pressure check (setupw.f90)
         - filter: Perform Action
           filter variables:
           - name: windEastward
           where:
           - variable: ObsType/windEastward
             is_in: 287
           action:
             name: inflate error
             inflation variable:
               name: ObsFunction/ObsErrorFactorPressureCheck
               options:
                 variable: windEastward
                 inflation factor: 4.0
                 SetSfcWndObsHeight: true
                 AddObsHeightToStationElevation: true
                 AssumedSfcWndObsHeight: 10

         # Error inflation based on pressure check (setupw.f90)
         - filter: Perform Action
           filter variables:
           - name: windNorthward
           where:
           - variable: ObsType/windNorthward
             is_in: 287
           action:
             name: inflate error
             inflation variable:
               name: ObsFunction/ObsErrorFactorPressureCheck
               options:
                 variable: windNorthward
                 inflation factor: 4.0
                 SetSfcWndObsHeight: true
                 AddObsHeightToStationElevation: true
                 AssumedSfcWndObsHeight: 10

```

Reply:
Are these new options part of your PR for UFO?